skbio/alignment/_tabular_msa.py

```diff
@@ -317,6 +339,43 @@ def __init__(self, sequences, metadata=None, positional_metadata=None,
         PositionalMetadataMixin._init_(
             self, positional_metadata=positional_metadata)
 
+        # Set up our indexers
+        self._loc = TabularMSALoc(self)
+        self._iloc = TabularMSAILoc(self)
+
+    def _constructor_(self, sequences=NotImplemented, metadata=NotImplemented,
+                      positional_metadata=NotImplemented,
+                      index=NotImplemented):
+        """Return new copy of the MSA with overridden properties.
+
+        NotImplemented is used as a sentinel so that None may be used to
+        override values.
+        """
+
+        if sequences is NotImplemented:
+            sequences = self._seqs
```

Comment:
Should sequences get shallow-copied in the constructor? `metadata` and `positional_metadata` are being shallow-copied in the constructor and `index` is immutable but sequences aren't being copied.